pkg/3scale/amp/operator/apicast_reconciler.go

```diff
@@ -51,6 +51,9 @@ func (r *ApicastStagingDCReconciler) IsUpdateNeeded(desired, existing *appsv1.De
 	tmpUpdate = DeploymentConfigReconcileContainerResources(desired, existing, r.Logger())
 	update = update || tmpUpdate
 
+	tmpUpdate = DeploymentConfigReconcileDeploymentTriggerImageChangeParamsName(desired, existing, r.Logger())
```

Comment:
We reconcile the image change trigger, specifically the name of it. I'm unsure on whether we should reconcile this. What do you think, should we reconcile ImageChange trigger names in the DCs? It's not directly related to a CR parameter but at the same time makes sense to make sure it has the correct version set. The same situation more or less happens with what we are reconciling on our ImageStreams.

Reply:
IMO we should not reconcilliate imageChangeTrigger. We do not reconcilliate either `ampRelease` (which has the same value) for system deployments. 

We reconcilliate imagestreams because images are exposed in the CR.

Reply:
So we leave the reconciliation of the imageChangeTrigger to upgrade-only right? Because we have to change it when a new 3scale release (non-cve) appears

Reply:
Yes, we need to upgrade, but not reconcilliate. I do not see any reason to reconcilliate that part of the deployment config.